.github/workflows/makefile.yml

```
@@ -13,6 +13,7 @@ jobs:
     - uses: actions/checkout@v4
     - name: Install tools used in tests
       run: |
+        apt install -y just
```

```suggestion
        sudo apt install -y just
```
This should fix the CI (I think).